content/starter-configs/code-snippets/diffy-integration.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Diffy Integration"
+date: 2020-07-17T17:00:00-04:00
+weight: 3
+---
+
+Tugboat does not provide `visualdiffs` for pages behind authentication. If you need to generate visual diffs from pages
+that are protected behind a login screen, you can set up [Diffy](https://diffy.website/) for your Tugboat Previews.
+Diffy is a third-party visual diffing tool.
+
+## Prerequisites
+
+1. You'll need to set up the project in Diffy, and get the numeric project ID from Diffy.
+2. You'll need a Diffy API key to communicate with Diffy.
+
+## Tugboat Repository Configuration
+
+Create Tugboat [environment variables](/setting-up-services/how-to-set-up-services/custom-environment-variables/) to
+store your `DIFFY_API_KEY` and `DIFFY_PROJECT_ID`:
+
+![Example DIFFY_API_KEY and DIFFY_PROJECT_ID as Tugboat environment variables](/_images/diffy-envvars.png)
```

Comment:
All of our other screenshots are in light mode, maybe redo this one and with some dummy data? (I did de-activate that API key, though)

Reply:
Good point! I'll do that.